src/devices/infactory.c

```diff
@@ -92,17 +135,43 @@ static char *output_fields[] = {
         "channel",
         "temperature_F",
         "humidity",
+        "mic",
+        "battery",
+#ifdef DEBUG        
+        "rawmsg",
+#endif // DEBUG        
         NULL,
 };
 
+/*
+Analysis using Genuino (see http://gitlab.com/hp-uno, e.g. uno_log_433):
+
+Observed On-Off-Key (OOK) data pattern:
+
+    preamble            syncPrefix        data...(40 bit)                        syncPostfix
+    HHLL HHLL HHLL HHLL HLLLLLLLLLLLLLLLL (HLLLL HLLLLLLLL HLLLL HLLLLLLLL ....) HLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLLL
+
+Breakdown:
+
+- four preamble pairs '1'/'0' each with a length of ca. 1000uS
+- syncPre, syncPost, data0, data1 have a '1' start pulse of ca. 500uS
+- syncPre pulse before dataPtr has a '0' pulse length of ca. 8000uS
+- data0 (0-bits) have then a '0' pulse length of ca. 2000uS
+- data1 (1-bits) have then a '0' pulse length of ca. 4000uS
+- syncPost after dataPtr has a '0' pulse length of ca. 16000uS (we declare it received at 12ms)
+
+This analysis is the reason for the new r_device definitions below.
+NB: pulse_demod_ppm does not use .gap_limit if .tolerance is set.
```

Comment:
If the old samples decode then I think it is ok.

Reply:
I checked it with my own samples. Will look for the original ones (they are probably still on-site).

Reply:
Checked OK. :-)